index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <form id="recipe-form" type="submit">
+    <form id="recipe-form" type="add recipe">
```

Comment:
```suggestion
    <form id="recipe-form">
```
`type` is not a valid attribute for a `<form>` element.